index.js

```diff
@@ -65,7 +65,8 @@ function photon (imageUrl, opts) {
     if (parsedUrl.search) {
       return null;
     }
-    params.pathname = url.format( parsedUrl ).substring(1);
+    var formattedUrl = url.format( parsedUrl );
+    params.pathname = formattedUrl.startsWith( '//' ) ? formattedUrl.substring(1) : formattedUrl;
```

Comment:
> TypeError: Object //example.com/image.png has no method 'startsWith'

If we really do support 0.8, 0.10, and Node 0.12 we'll need to find an alternative. Perhaps a simple regex will do?
```js
/^\/\//.test( formatedUrl ) ? formattedUrl.substring(1) : formattedUrl;
```

Reply:
Tried and true `formattedUrl.indexOf( '//' ) === 0` should be synonymous.

Reply:
Ah that's much simpler :D Let's go with @sirreal's suggestion.

Reply:
Damn, I never worked with node versions so ancient, it's like going back to IE :)

Yeah, I didn't even conceive a version of JS without `startsWith`, `indexOf` it's better.